ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14-periodics.yaml

```diff
@@ -5438,18 +5416,7 @@ periodics:
     ci-operator.openshift.io/variant: amd64-nightly-4.14-upgrade-from-stable-4.14
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-4.14-upgrade-from-stable-4.14-baremetalds-agent-hypershift-mce-inplace-f28
-  reporter_config:
-    slack:
-      channel: '#forum-prow-hypershift-qe-ci'
```

Comment:
Slack notifications are unexpected removed ?

Reply:
It was removed by `make update`, I tried again, couldn't add it back, so I added it manually.

Reply:
Fixed

Reply:
Maybe you need a double check ?

Reply:
Done